deploy/deploy.sh

```diff
@@ -5,8 +5,8 @@ set -x
 hco_namespace=kubevirt-hyperconverged
 
 # Create the namespaces for the HCO
-kubectl create ns $hco_namespace
-
+kubectl create ns ${hco_namespace}
+kubectl config set-context --current --namespace=${hco_namespace}
```

Comment:
There's a reason that command was removed last time - we don't want the context to be changed to `kubevirt-hyperconverged` namespace after the deployment is done. instead, we have `-n ${hco_namespace}` on every namespaced resource below.